products/health-care/appointments/va-online-scheduling/express-care-research/express-care-conversation-guide.md

```diff
@@ -0,0 +1,116 @@
+  - If yes: **Once I start recording, I am going to confirm that you are ok with me recording this session once more.** 
+
+*Start recording.*
+
+- **I have started recording. I'd like to confirm: Are you comfortable if I record my screen the audio as we talk today?**
+
+## Section 0: Warm-up questions
+
+Before we look at the website, let’s start with a few warm-up questions.
+
+- Have you heard of "Express Care"?
+  - If YES: 
+    - What is it? 
+    - Have you used it?
+  - If NO: What do you think it might be?
+  - Prompt if needed: Is it Telehealth or in-person?
+- Are you aware of any forms of Telehealth appointments offered by VA providers?
+  - If YES: 
+    - Which ones?
+  - Have you ever utilized these services? 
+    - (Prompt if needed: VA or non-VA providers)
+  - How did you become aware of them?
+
+## Section 1: Value of VA urgent Telehealth
+
+- Have you ever used Telehealth appointments?
+  - If YES: 
+    - Was this with a VA provider or non-VA doctor?
+        - If VA: How did you schedule your appointment 
+    - Was this a same-day appointment?
+    - Can you tell me a bit more about your experience?
+  - If NO: 
+    - In your own words, can you define "Telehealth"?
+    - Have you considered it before? 
+    - What would convince you to try it?
+  - Would you consider using Telehealth appointments for a same-day appointment?
+- Have you heard anything about same-day/urgent care Telehealth appointments through VA?
+  - (Clarification question if needed: Do you think that VA offers any same-day urgent care appointments? Do you think that the VA offers any same-day urgent care Telehealth appointments?)
+- Have you ever tried to make a same-day appointment with a VA provider, either someone you see regularly or any available provider?
+  - If YES: 
+    - Can you tell me what happened?
+    - How did you make this appointment?
+    - Were you able to be seen?
+  - If NO:
+    - Have you ever considered doing this, but something prevented you? If so, what?
+
+## Section 2: Scheduling expectations
+
+- Do you consider "same-day" or urgent care the same thing as scheduling an appointment? If not, how is this different?
+- Can you tell me what types of urgent care you might seek through VA's same-day Telehealth appointments?
+- Who do you expect to speak with for an urgent same-day VA Telehealth appointment?
+
+## Section 3: Coronavirus appointments 
+
+Next, I'm going to ask you some questions related to coronavirus. These are general questions related to how you might seek medical help if you are experiencing symptoms. If you feel uncomfortable at any point we can move on to the next section or stop, just let me know. 
+
+- When you are speaking with family or friends how do you refer to the current pandemic?
+- Have you been practicing physical distancing, physical isolation, or quarantined, by choice or otherwise?
```

Comment:
For the first 2 questions in this section (refer and physical distancing) feel out of place given the context of this script/research with the user. I am not sure I understand how data/info provided to those two questions helps this research effort. Perhaps it's to help the VA more broadly about how to talk about the pandemic?

Reply:
@lenaecb these are questions USDS asked all COVID-19 research participants. I included them in part for consistency, but also because the VA team is trying to learn a bit more for content. We can also use these to build rapport/warm ups to get the participant comfortable talking about the coronavirus situation as well as learn from our target users (Veterans) how they think of the pandemic.

We had some more robust/probing questions in the research scripts for CDC, so I've incorporated them into the script. I don't want to spend too much time on this in the session, but it's a good learning opportunity